extensions/ql-vscode/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## [UNRELEASED]
 
 - Fix the _CodeQL: Open Referenced File_ command for Windows systems. [#979](https://github.com/github/vscode-codeql/pull/979)
+- Fix the _CodeQL: Open Referenced File_ command for Windows systems. [#979](https://github.com/github/vscode-codeql/pull/979) 
+- Fix a bug that causes VSCode to crash when handling large SARIF files (>4GB) [#1004](https://github.com/github/vscode-codeql/pull/1004)
```

Comment:
I would rephrase this more proactively. You've rewritten it to overcome a limitation of Node.js defaults, rather than fixing a bug in our handling.
```suggestion
- Support large SARIF results files (>4GB) without crashing VS Code [#1004](https://github.com/github/vscode-codeql/pull/1004)
```